### PR TITLE
ItineraryPoint

### PR DESCRIPTION
**Pull request by PFGimenez**
Written with vim and not tested (but it compiles)

### CODE REVIEW

File: sources/struct/itinerary_point.cpp [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#include "itinerary_point.h"
+#include <math.h>
```
Comment (bourreP) [inline sources/struct/itinerary_point.cpp]: you shouldn't use the lib c, and prefer cmath

File: sources/struct/itinerary_point.cpp [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    ItineraryPoint::ItineraryPoint(const float &x, const float &y,
+            const float &orientation, const float &curvature,
+            const bool &going_forward, const float &max_speed,
+            const float &possible_speed, const bool &stop) :
+        x_(x), y_(y),
+        orientation_(orientation), curvature_(curvature),
+        going_forward_(going_forward), max_speed_(max_speed),
+        possible_speed_(possible_speed), stop_(stop)
+    {
+        orientation_ = fmod(orientation_, 2 * M_PI);
```
Comment (bourreP) [inline sources/struct/itinerary_point.cpp]: with cmath, replace fmod with std::fmod

**Comment by Hindi** [comment]
You should surround streams in `#if DEBUG` preprocessor directives.

**Comment by Hindi** [comment]
`#include <ostream>` also need to be used in `DEBUG` only.

**Pull request merged**